packages/input-option/src/InputOption/InputOption.types.ts

```diff
@@ -3,6 +3,24 @@ import { PropsWithChildren } from 'react';
 import { AriaLabelProps } from '@leafygreen-ui/a11y';
 import { DarkModeProps } from '@leafygreen-ui/lib';
 
+const ActionType = {
+  Default: 'default',
+  Destructive: 'destructive',
+} as const;
+
+type ActionType = (typeof ActionType)[keyof typeof ActionType];
+
+export { ActionType };
+
+const RenderedContext = {
+  FormElement: 'formElement',
+  Menu: 'menu',
+} as const;
```

Comment:
Can we a bit more terse here? 
```
const RenderedContext = {
  Form: 'form',
  Menu: 'menu',
} as const;
```

Reply:
Also, can we add some comments on what this means, and what it changes about the component?

Reply:
yes